src/CMakeLists.txt

```diff
@@ -4,19 +4,31 @@ project(serlio CXX)
 
 include(common.cmake)
 
+# Enable solution folders for Visual Studio
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
```

Comment:
a bit too prominent, as it only affects the visual studio generator... suggest to hide it somewhere

Reply:
fixed